spec/01-unit/011-balancer_spec.lua

```diff
@@ -28,14 +30,54 @@ describe("Balancer", function()
       wait_max = 0.5,         -- max wait time before discarding event
     })
 
+    local hc_defaults = {
+      active = {
+        timeout = 1,
+        concurrency = 10,
+        http_path = "/",
+        healthy = {
+          interval = 0,  -- 0 = probing disabled by default
+          http_statuses = { 200, 302 },
+          successes = 0, -- 0 = disabled by default
+        },
+        unhealthy = {
+          interval = 0, -- 0 = probing disabled by default
+          http_statuses = { 429, 404,
+                            500, 501, 502, 503, 504, 505 },
+          tcp_failures = 0,  -- 0 = disabled by default
+          timeouts = 0,      -- 0 = disabled by default
+          http_failures = 0, -- 0 = disabled by default
+        },
+      },
+      passive = {
+        healthy = {
+          http_statuses = { 200, 201, 202, 203, 204, 205, 206, 207, 208, 226,
+                            300, 301, 302, 303, 304, 305, 306, 307, 308 },
+          successes = 0,
+        },
+        unhealthy = {
+          http_statuses = { 429, 500, 503 },
+          tcp_failures = 0,  -- 0 = circuit-breaker disabled by default
+          timeouts = 0,      -- 0 = circuit-breaker disabled by default
+          http_failures = 0, -- 0 = circuit-breaker disabled by default
+        },
+      },
+    }
+
+    local passive_hc = utils.deep_copy(hc_defaults)
```

Comment:
Should we get the defaults from the schema instead (like we do in the migrations iirc), so that if they ever change, these tests don't use stale defaults?

Reply:
otoh hardcoded values are more likely to break with changes, which (imho) is a good thing for tests since it forces you to review and update

Reply:
btw: in migrations we use hardcoded, not schema defaults. We tried that, but future schema updates will break past migrations if you do.